phpBB/includes/acp/acp_forums.php

```diff
@@ -1918,6 +1918,18 @@ function delete_forum_content($forum_id)
 
 		$table_ary = array(FORUMS_ACCESS_TABLE, FORUMS_TRACK_TABLE, FORUMS_WATCH_TABLE, LOG_TABLE, MODERATOR_CACHE_TABLE, POSTS_TABLE, TOPICS_TABLE, TOPICS_TRACK_TABLE);
 
+		/**
+		 * Perform additional actions before forum content deletion
+		 *
+		 * @event core.delete_forum_content_before_query
+		 * @var	array	table_ary	Array of tables from which all rows will be deleted that hold the forum_id
+		 * @since 3.1.5-RC1
+		 */
+		$vars = array(
+				'table_ary',
+		);
```

Comment:
In this function, only $forum_id is available, and it won't be of any real use for an extension in this context.